api/src/resources/application.yml

```diff
@@ -0,0 +1 @@
+admin.password=$2a$10$Ko74E4Zrz4kWwZ82vQIDLO4MH0yV5Dab8P5Cj5coWlmTAjsWFCHbK
```

Comment:
to nie jest poprawny yaml:)
1.Yamla zróbmy tak:
```
application:
  admin:
   password: "{bcrypt}value"
```
2.Zróbmy już jakieś trudniejsze hasło niż admin:)